project.clj

```diff
@@ -28,12 +29,15 @@
   :source-paths ["src/clj"]
   :java-source-paths ["src/java"]
 
-  :profiles {:dev {:dependencies [[puppetlabs/trapperkeeper ~tk-version
+  :profiles {:dev {:dependencies [[puppetlabs/trapperkeeper nil
                                    :classifier "test"
                                    :scope "test"]
-                                  [puppetlabs/kitchensink ~ks-version
+                                  [puppetlabs/kitchensink nil
```

Comment:
Yeah, the `nil` would signify that that the version is pulled in from the parent project.  It may actually work without the `nil` in later lein-parent versions, can't remember.  I can try removing it and see if it still works.